src/app/layout.tsx

```diff
@@ -1,21 +1,22 @@
-import './globals.css'
-import { Inter } from 'next/font/google'
+import "./globals.css";
+import { Inter } from "next/font/google";
 
-const inter = Inter({ subsets: ['latin'] })
+const inter = Inter({ subsets: ["latin"] });
 
 export const metadata = {
-  title: 'Create Next App',
-  description: 'Generated by create next app',
-}
+  title: "Faker: JSON Placeholder",
```

Comment:
Also mobile view is a bit janky
<img width="414" alt="CleanShot 2023-06-02 at 18 18 25@2x" src="https://github.com/debbyalmadea/faker/assets/14111227/e9149996-78a0-4566-b44d-e1715e62669d">

Reply:
fixed in latest commit, now the ui shouldn't be tacky in smaller screen